egs/spgispeech/ASR/zipformer/model.py

```diff
@@ -0,0 +1,358 @@
+# Copyright    2021-2023  Xiaomi Corp.        (authors: Fangjun Kuang,
```

Comment:
Could we symbol link this file to existing one?

Reply:
Yes, will change it.